clio-client/src/main/scala/org/broadinstitute/clio/client/commands/MoveWgsUbamCommand.scala

```diff
@@ -31,63 +28,96 @@ object MoveWgsUbamCommand
+        "An error occurred while copying the files in google cloud. No files have been moved."
       upsertUbam <- upsertUpdatedWgsUbam(webClient, config) withErrorMsg
         """An error occurred while upserting the WgsUbam. The state of clio is now inconsistent.
           |The ubam exists in both at both the old and the new locations.
           |At this time, Clio only knows about the bam at the old location.
           |Then, try upserting (not moving) the unmapped bam with the correct path then deleting the old bam.
-          |If the state of Clio cannot be fixed, please contact the Green Team at greenteam@broadinstitute.org
+          |If the state of Clio cannot be fixed, please contact the Green Team at greenteam@broadinstitute.org.
         """.stripMargin
-      _ <- deleteGoogleObject(wgsUbam.ubamPath, ioUtil) withErrorMsg
+      _ <- deleteGoogleObject(wgsUbamPath, ioUtil) withErrorMsg
         """The old bam was not able to be deleted. Clio has been updated to point to the new bam.
-          | Please delete the old bam. If this cannot be done, contact Green Team at greenteam@broadinstitute.org
+          | Please delete the old bam. If this cannot be done, contact Green Team at greenteam@broadinstitute.org.
           | """.stripMargin
     } yield {
+      logger.info(
+        s"Successfully moved '${wgsUbamPath.get}' to '${config.ubamPath.get}'"
+      )
       upsertUbam
     }
   }
 
-  private def queryForWgsUbam(
-    webClient: ClioWebClient,
-    config: BaseArgs
-  ): Future[TransferWgsUbamV1QueryOutput] = {
-    val httpResponse = webClient.queryWgsUbam(
-      config.bearerToken.getOrElse(""),
-      TransferWgsUbamV1QueryInput(
-        flowcellBarcode = config.flowcell,
-        lane = config.lane,
-        libraryName = config.libraryName,
-        location = Option(Location.GCP),
-        documentStatus = Option(DocumentStatus.Normal)
-      )
-    )
+  private def verifyPath(webClient: ClioWebClient,
+                         config: BaseArgs): Future[Unit] = {
+    implicit val executionContext: ExecutionContext = webClient.executionContext
+    Future(config.location.foreach {
+      case "GCP" => ()
+      case _ =>
+        logger.error("Only GCP unmapped bams are supported at this time.")
+        throw new Exception()
+    }).flatMap { _ =>
+      Future(config.ubamPath.foreach {
+        case loc if loc.startsWith("gs://") => ()
+        case _ =>
+          logger.error(
+            s"The destination of the ubam must be a cloud path. ${config.ubamPath.get} is not a cloud path."
+          )
+          throw new Exception()
+      })
+    }
+  }
+
+  private def queryForWgsUbamPath(webClient: ClioWebClient,
+                                  config: BaseArgs): Future[Option[String]] = {
     implicit val ec: ExecutionContext = webClient.executionContext
+
+    def ensureOnlyOneJson(wgsUbams: Json): Json = {
+      wgsUbams.asArray.getOrElse(throw new Exception()).size match {
+        case 1 =>
```

Comment:
It looks like all uses of `1` here refer to the same concept. Replacing them all with a named constant would make that clear.

Reply:
I'm not sure storing 1 in a named variable buys us much here. Its just the size of the list

Reply:
There are three `1`s here that refer to the same concept; the list size is not the the concept; the limit is. Changing the code to support 1 and 0 cases could also get rid of this issue.
```
match {
case 1 => wgsUbams.asArray.get.head
case 0 => throw new Exception(...)
case _ => throw new Exception(...)
}
```

Reply:
Good point. Changed